addons/point_of_sale/point_of_sale_view.xml

```diff
@@ -943,7 +954,23 @@
             </field>
         </record>
 
-        <menuitem action="action_pos_session_opening"
+        <record model="ir.actions.act_window" id="action_pos_config_kanban">
+            <field name="name">Point of Sale</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">pos.config</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">kanban,tree,form</field>
+            <field name="search_view_id" ref="view_pos_config_search" />
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click to create a new PoS config.
+              </p><p>
+                A PoS config allows you to sell through the PoS interface.
```

Comment:
Change string into : Configure at least one Point of Sale to be able to sell through the PoS interface.